### PR TITLE
Update header css

### PR DESCRIPTION
**Pull request by siapankina**
Screenshot 2020-11-11 at 17 09 08
Screenshot 2020-11-11 at 17 09 31

### CODE REVIEW

File: unboxed/static/css/unboxed.css [hunk 1]
```diff
@@ -23,12 +23,14 @@ body, wrapper {
  }
 
 .navbar {
-  background-color: #ffffff;
+  background: linear-gradient(130.11deg, #B54C7B 0%, #913487 27.69%, #510B81 90.11%);
 }
 .navbar .nav-link {
   font-size: 14px;
+  font-family: Work Sans;
```
Comment (melzreal) [inline unboxed/static/css/unboxed.css:30]: so the font just looks completely wrong. I know it says its the right family, but if we compare them side by side its just not right, it needs to look like so: 
image

Comment (melzreal) [reply unboxed/static/css/unboxed.css:30]: it also needs extra padding on the left of the unboxed, and maybe reducing the pixel size in the header could help with the blur 🤔

Comment (siapankina) [reply unboxed/static/css/unboxed.css:30]: do you know which font-family is that?

Comment (melzreal) [reply unboxed/static/css/unboxed.css:30]: https://fonts.google.com/specimen/Work+Sans?sidebar.open=true&selection.family=Work+Sans:wght@500

Also sans-serif
font-family: 'Work Sans', sans-serif;

Comment (melzreal) [reply unboxed/static/css/unboxed.css:30]: you can go to the fonts link and download them - we should use the woff files (you can see the Roboto files for ex are already installed in the app) so download the needed ones from google fonts then add them to the fonts folder, then reference with the weight and right seriff - i.e in unboxed.css you have the example

```  src: local('Roboto'), url('../fonts/Roboto/Roboto-Regular.ttf') format('truetype');```

so you'd target the specific woff file for the font

**Review by melzreal — APPROVED** [review]
🚀 🚀  👯

**Pull request merged**